data/helpers.d/network

```diff
@@ -0,0 +1,15 @@
+# Find a free port and return it
+#
+# example: port=$(ynh_find_port 8080)
+#
+# usage: ynh_find_port begin_port
+# | arg: begin_port - port to start to search
+ynh_find_port () {
+	port=$(echo "$1" | sed 's/[^0-9]//g')	# Eliminate all non-digit characters
```

Comment:
IMHO the function should fail and generate an error if the argument is not a valid port number